src/types/ilp-logger/index.d.ts

```diff
@@ -0,0 +1,20 @@
+/// <reference types="node" />
+
+declare module 'ilp-logger' {
```

Comment:
If you've created a type file for the logger, you can use the `Logger` type instead of the `any` type for it

Reply:
Would it still work in the event the plugin is using riverpig? That might break the typing unless we import the type from riverpig as 'riverLogger' for example and set `this._log: Logger | riverLogger`

Reply:
hmn, that's a good point. You can leave it as-is for now but maybe we could make an interface for riverpig-like loggers and just say the plugin uses anything that complies to that